docs/dashboards/custom-component.md

```diff
@@ -0,0 +1,78 @@
+Custom Dashboards components
+===
+This article shows how to create custom Dashboards Component, in this example YouTube Component.
+
+Note, that to create the custom component we are using ES6, to use the `class` and `extends` keywords, which makes creating custom class much easier.
+
+We start by importing the default `Component` class from `Dashboards` namespace. The next step is creating the class which will inherit from the imported `Component` class.
+The name of the class will automatically be the string, that will be used to reference this component type e.g. the class `YouTubeComponent` will be referenced by name: "`YouTube`".
+
+```js
+    const Component = Dashboards.Component;
+
+    class YouTubeComponent extends Component {
+
+    }
+```
+
+Then, depending on what the Component is expected to do, the options are limitless. In this example, one `iframe` element will be added, which will accept one attribute from options, which is `videoId`, and since the iframe element needs it size to be defined, the resize method is extended to update ths size of the element.
+
+At the end, the new YouTubeComponent class needs to be addded to the registry, using the `Component.addComponent` method.
```

Comment:
Please wrap the YouTubeComponent into brackets, in will increase the readibility.